tests/unit/logicblocks/event/store/adapters/test_postgres.py

```diff
@@ -1,13 +1,26 @@
 import sys
+from datetime import datetime, timezone
+from typing import cast
+from unittest.mock import AsyncMock
 
 import pytest
-from psycopg import AsyncConnection
+from psycopg import AsyncConnection, sql
 from psycopg_pool import AsyncConnectionPool
 
 from logicblocks.event.db import PostgresConnectionSettings
 from logicblocks.event.store.adapters import (
     PostgresEventStorageAdapter,
 )
+from logicblocks.event.store.adapters.postgres import (
+    StoredEvent,
+    TableSettings,
+    insert_batch,
+    insert_batch_query,
+)
```

Comment:
We should always import from one of the top level packages, i.e., one of:
```python
logicblocks.event.db
logicblocks.event.processing
logicblocks.event.projection
logicblocks.event.sources
logicblocks.event.store
logicblocks.event.testing
logicblocks.event.types
logicblocks.event.utils
```

I realise this code is already breaking this rule but I think I've fixed it in another PR that has already been merged.